splunk_connect_for_snmp_poller/manager/mib_server_client.py

```diff
@@ -102,3 +103,14 @@ def format_value_for_mib_server(value, value_type):
         return value.prettyPrint()
     else:
         return str(value)
+
+
+def get_mib_profiles():
+    mib_server_url = os.environ["MIBS_SERVER_URL"]
+    endpoint = "profiles"
+    profiles_url = os.path.join(mib_server_url.strip("/"), endpoint)
+
+    try:
+        return requests.get(profiles_url, timeout=3).text
+    except Exception:
+        return {}
```

Comment:
we should add logger.exception("..") here
it logs given message + stacktrace, additionally we can catch different exceptions as it is well known what requests can throw to make our lifes in the future easier